newsfragments/1526.feature.rst

```diff
@@ -0,0 +1,2 @@
+Some users are confused with the usage of `open_signal_receiver <https://trio.readthedocs.io/en/stable/reference-io.html#trio.open_signal_receiver>`__ being able to operate without any signal.
```

Comment:
Just say
```
`open_signal_receiver`
```
and Sphinx will resolve the link for you automatically.

Reply:
I followed this example https://raw.githubusercontent.com/python-trio/trio/master/newsfragments/1241.feature.rst

Reply:
(It also makes sure the function actually exists, which is handy to avoid dead links.)